assets/docs.html

```diff
@@ -150,33 +147,34 @@ <h2 class="docs-subtitle">Contributors</h2>
 					<div class="span4">
 						<div class="footer-title">ABOUT US</div>
 						WindBot is a high quality automation and enhancement software that functions alongside the Tibia client. It works by sending mouse clicks and keyboard strokes, displaying it's own visuals on top, facilitating player communication and is customizable in many many ways.
-						<br><br>
+						<br /><br />
 						It was created by Lucas Terra, one of the main minds behind the success of NeoBot and ElfBot. It took several months of work and tests before...
-						<br><br>
+						<br /><br />
 						<a href="about-us.html" class="blue">Read more »</a>
 					</div>
 					<div class="span4">
 						<!-- Sections -->
 						<div class="footer-title">SECTIONS</div>
-						<a href="news.html">» News</a><br><br>
-						<a href="features.html">» Features</a><br><br>
-						<a href="download.html">» Download</a><br><br>
-						<a href="purchase.html">» Purchase</a><br><br>
-						<a href="lua-variables.html">» Documentation</a><br><br>
+						<a href="news.html">» News</a><br /><br />
+						<a href="features.html">» Features</a><br /><br />
+						<a href="download.html">» Download</a><br /><br />
+						<a href="purchase.html">» Purchase</a><br /><br />
+						<a href="lua-variables.html">» Documentation</a><br /><br />
```

Comment:
I agree the ideal goal is to have them managed by npm and have some automated concatenation and minification, not only for performance reasons but also for security and ease of use. 

However, that would mean that, in order to make sure we're using the same files on the whole website, we'd have to host WindBot's entire website on a repository and use the same build system we think of here, and I just don't think we're ready to start messing with any other part of the website right now.

*PS:* I'm sorry if my comment about the `<br />` tags sounded condescending. It's just that you implied to be new on open source contributions and I thought I'd give you a heads up.